stable/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Should be namespaced as well: `{{- define "newrelic-infra.name" -}}`

Reply:
👍 fixed